ui/page/seedbackup/backup_instructions.go

```diff
@@ -120,6 +120,11 @@ func (pg *BackupInstructionsPage) OnNavigatedFrom() {}
 // to be eventually drawn on screen.
 // Part of the load.Page interface.
 func (pg *BackupInstructionsPage) Layout(gtx C) D {
+	for i := range pg.checkBoxes {
+		if pg.checkBoxes[i].CheckBox.Update(gtx) {
+			pg.ParentWindow().Reload()
+		}
+	}
```

Comment:
I don't think this is best solution, we only need to enable/disable `View seed phrase` button when all boxes checked

Reply:
It was added to fix the button not active after I check all checkbox until somewhere on the screen is touched